odoo/addons/base/models/ir_asset.py

```diff
@@ -68,7 +68,8 @@ def create(self, vals_list):
         return super().create(vals_list)
 
     def write(self, values):
-        self.env.registry.clear_cache('assets')
+        if self:
```

Comment:
Do you still need this change? Seems like it was needed in an intermediate state of the PR where you could write on empty record set (because of a filtered), no?

Reply:
Yes I could remove it I think but anyway it makes sense.